TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst

```diff
@@ -716,8 +718,8 @@ Example::
 
 	logoUri = TYPO3.Neos:ImageUri {
 		asset = ${q(node).property('image')}
-		maximumWidth = 100
-		maximumHeight = 100
+		width = 100
+		height = 100
```

Comment:
In fact as far as I know cropping cropping needs this since 2.0 to actually work.